Apply -Wunusual-code to spdif_tx_reconfig_port()

xross opened the pull request.

ed-xmos commented on lib_spdif/src/SpdifTransmitReconfigPort.xc:
```diff
@@ -0,0 +1,16 @@
+#include <xs1.h>
+#include "spdif.h"
+
+/* Note, this function is in a seperate fill such that -Wunusual-code can be applied to it */
```
typo - fill -> file

ed-xmos reviewed (changes requested):
BOOM! Happy to see this. Maybe a changelog entry makes sense?
FIXED - warning about legal use of resource pointe

xross commented:
> BOOM! Happy to see this. Maybe a changelog entry makes sense? FIXED - warning about legal use of resource pointe



> BOOM! Happy to see this. Maybe a changelog entry makes sense? FIXED - warning about legal use of resource pointe

Done :)

ed-xmos reviewed (approved):
Great!
